### PR TITLE
data:merwinmathew33

### PR DESCRIPTION
**Pull request by merwinmathew33**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/4105"></a>

### CODE REVIEW

**Review by AbbeyIT — APPROVED** [review]
LGTM! Thank you for your contribution.

**Comment by merwinmathew33** [comment]
My json check is failed
Can you tell me the reason?

File: data/merwinmathew33.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
+{
+  "name": "Merwin Mathew",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "Engineering student who is passionate about **DevOps** and **open source**",
+  "avatar": "https://github.com/merwinmathew33.png"
```
Comment (swagataroy30) [inline data/merwinmathew33.json:6]: add comma in line 6

**Comment by SaraJaoude** [comment]
I am closing this PR as this appears to be a duplicate of: https://github.com/EddieHubCommunity/LinkFree/pull/4114

You can append more commits to an open PR but if you do not wish to do so and want to open a new one, please close the PR that you no longer want as this is helpful to maintainers.

**Pull request closed without merging**